modules/end-user-guide/pages/selecting-an-ide.adoc

```diff
@@ -1,126 +1,35 @@
+:_content-type: ASSEMBLY
 :navtitle: Selecting an IDE
 :keywords: user-guide, selecting, IDE
 :page-aliases: che-theia-ide-basics, version-control, using-alternative-ides-in-che, configuring-a-workspace-to-use-an-ide-based-on-the-intellij-platform, building-images-for-ides-based-on-the-intellij-platform, provisioning-the-jetbrains-offline-activation-code, support-for-theia-based-ides, che-theia-troubleshooting, differences-in-how-che-theia-webview-works-on-a-single-host-mode-comparing-to-a-multi-host-mode
 
 [id="selecting-an-ide_{context}"]
 = Selecting an IDE
 
-link:https://theia-ide.org/docs/[Eclipse Theia] is the current default editor for {prod-short} but other IDEs are available when starting a workspace. 
+:context: selecting-an-ide
 
-The following alternatives are available:
+The default editor for {prod-short} is link:https://theia-ide.org/docs/[Eclipse Theia] but you can select a different IDE when starting a workspace. Following alternatives to Eclipse Theia are available:
 
-* JetBrains link:https://www.jetbrains.com/help/idea/getting-started.html[IntelliJ IDEA]
-* JetBrains link:https://www.jetbrains.com/help/pycharm/quick-start-guide.html[PyCharm]
+* JetBrains IDE such as link:https://www.jetbrains.com/help/idea/getting-started.html[IntelliJ IDEA] or link:https://www.jetbrains.com/help/pycharm/quick-start-guide.html[PyCharm]
 * Visual Studio Code - link:https://github.com/Microsoft/vscode/[Open Source ("Code - OSS")]
+* Your own IDE defined with the `inline` directive
```

Comment:
```suggestion
* An IDE that you defined by using the `inline` directive
```

- Also, consider keeping this as a list item only if the user selects this IDE like any other one when they start a workspace. Otherwise, if the user doesn't *select* this IDE but must instead *specify an inline directive* at this stage, consider making this phrase a separate non-list item.